### PR TITLE
chore: add a security policy

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

Created a security policy for our tool

fixes: APE-377

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Any expected deviations and necessary clarifications around the standard are explained in the following sections.
+
+## Disclosing a Vulnerability
+
+### Supported Versions
+
+Ape is currently under active development, targeting a 1.x release.
+Any security issues found on the latest minor version of Ape will be fixed and released in either
+a patch or minor release.
```
Comment (unparalleled-js) [inline SECURITY.md:34]: The formatting of this document is veering from our norm of always doing 1 line per sentence in markdown.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                    | Keybase                                         |
+| ------------------------------------------ | ----------------------------------------------- |
+| [@fubuloubu](https://github.com/fubuloubu) | [@fubuloubu](https://keybase.io/fubuloubu/chat) |
+
+## Sending Disclosures
+
+In the case where we become aware of security issues affecting projects using Ape,
+our intention is to engage in responsible disclosures with them as described in the adopted
```
Comment (unparalleled-js) [inline SECURITY.md:61]:
```suggestion
our intention is to engage in responsible disclosures with them as described in the adopted
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Security Policy
+
+This document describes the Security Policy for ApeWorX Ltd.,
+including vulnerability disclosures for the [Ape Framework](https://github.com/ApeWorX/ape)
+and any of it's [maintained plugins](https://github.com/ApeWorX?q=%22ape-%22&type=public&language=&sort=).
```
Comment (unparalleled-js) [inline SECURITY.md:5]: I don't really have a better answer yet, nor is this a huge deal, but this query is not 100% right. It shows a few other repos besides plugins in it.

**Review by unparalleled-js — APPROVED** [review]

File: SECURITY.md (hunk 3 above)
Comment (NotPeopling2day) [reply SECURITY.md:5]: Still could be improved https://github.com/search?q=org%3Aapeworx+ape-&type=repositories

Comment (unparalleled-js) [reply SECURITY.md:5]: For some reason, the docs site repo shows in that list

Comment (fubuloubu) [reply SECURITY.md]: Found a better one

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+ApeWorX is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                    | Keybase                                         |
+| ------------------------------------------ | ----------------------------------------------- |
+| [@fubuloubu](https://github.com/fubuloubu) | [@fubuloubu](https://keybase.io/fubuloubu/chat) |
```
Comment (fubuloubu) [inline SECURITY.md:46]:
```suggestion
| Contact                                              | Keybase                                                   |
| ---------------------------------------------------- | --------------------------------------------------------- |
| [@fubuloubu](https://github.com/fubuloubu)           | [@fubuloubu](https://keybase.io/fubuloubu/chat)           |
| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notepeopling2day/chat) |
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+ApeWorX is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                              | Keybase                                                   |
+| ---------------------------------------------------- | --------------------------------------------------------- |
+| [@fubuloubu](https://github.com/fubuloubu)           | [@fubuloubu](https://keybase.io/fubuloubu/chat)           |
```
Comment (fubuloubu) [inline SECURITY.md:46]:
```suggestion
| Contact                                               | Keybase                                                    |
| ----------------------------------------------------- | ----------------------------------------------------------- |
| [@fubuloubu](https://github.com/fubuloubu)            | [@fubuloubu](https://keybase.io/fubuloubu/chat)             |
```

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+ApeWorX is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                               | Keybase                                                    |
+| ----------------------------------------------------- | ----------------------------------------------------------- |
+| [@fubuloubu](https://github.com/fubuloubu)            | [@fubuloubu](https://keybase.io/fubuloubu/chat)             |
```
Comment (fubuloubu) [inline SECURITY.md:46]:
```suggestion
| Contact                                                | Keybase                                                     |
| ------------------------------------------------------ | ------------------------------------------------------------ |
| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)              |
```

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+### Directly to ApeWorX
+
+ApeWorX is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                                | Keybase                                                     |
```
Comment (fubuloubu) [inline SECURITY.md:44]:
```suggestion
| Contact                                                | Keybase                                                      |
```

File: SECURITY.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
+# Security Policy
+
+This document describes the Security Policy for ApeWorX Ltd., including vulnerability disclosures for the [Ape Framework](https://github.com/ApeWorX/ape) and any of it's [maintained plugins](https://github.com/search?q=org%3Aapeworx+ape-+-repo%3AApeWorX%2Fape++-repo%3AApeWorX%2Fapeworx.github.io&type=Repositories).
```
Comment (unparalleled-js) [inline SECURITY.md:3]: I just realized - should we have a policy that includes all repos we maintain, including `ethpm-types` and all that? If so, where would that live?

File: SECURITY.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+We are committed to conduct our Security Process in a professional and civil manner.
+Public shaming, under-reporting, or misrepresentation of vulnerabilities will not be tolerated.
```
Comment (unparalleled-js) [inline SECURITY.md:5]: Are there consequences? What does this mean?

File: SECURITY.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+- [Setting Dates](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#setting-dates):
+  how to agree on timelines for releasing updates and making details of the issue public.
+
+Any expected deviations and necessary clarifications around the standard are explained in the following sections.
+
+## Disclosing a Vulnerability
+
+### Supported Versions
+
+Ape is currently under active development, targeting a 1.x release.
```
Comment (unparalleled-js) [inline SECURITY.md:29]: Is there any way to say this (and feel free to disagree) without having to remember to come back and update it once we hit 1.0?

File: SECURITY.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
+# Security Policy
```
Comment (unparalleled-js) [inline SECURITY.md:1]: Should we link to anything official regarding what this is? Perhaps https://csrc.nist.gov/glossary/term/security_policy

File: SECURITY.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+ApeWorX is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                                | Keybase                                                      |
+| ------------------------------------------------------ | ------------------------------------------------------------ |
+| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)              |
+| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notepeopling2day/chat) |
```
Comment (unparalleled-js) [inline SECURITY.md:47]: I feel left out. Do we need more people here? I volunteer if yes...

File: SECURITY.md (hunk 8 above)
Comment (fubuloubu) [reply SECURITY.md:3]: Could add that here I suppose

File: SECURITY.md (hunk 9 above)
Comment (fubuloubu) [reply SECURITY.md:5]: I don't know, might be more specific to smart contract exploits

File: SECURITY.md (hunk 10 above)
Comment (fubuloubu) [reply SECURITY.md:29]: Maybe
```suggestion
Ape is currently under active development, we support the latest release by major version.
```

File: SECURITY.md (hunk 12 above)
Comment (fubuloubu) [reply SECURITY.md:47]: Yes, but you need to add yourself to Keybase lol

Comment (unparalleled-js) [reply SECURITY.md:47]: My username is `antazoey`!

Comment (fubuloubu) [inline SECURITY.md:47]:
```suggestion
| Contact                                                | Keybase                                                      |
| ------------------------------------------------------ | ------------------------------------------------------------ |
| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)              |
| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notpeopling2day/chat) |
| [@unparalleled-js](https://github.com/unparalleled-js) | [@Antazoey](https://keybase.io/antazoey/chat) |
```

File: SECURITY.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                                | Keybase                                                      |
+| ------------------------------------------------------ | ------------------------------------------------------------ |
+| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)              |
+| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notpeopling2day/chat) |
+| [@unparalleled-js](https://github.com/unparalleled-js) | [@Antazoey](https://keybase.io/antazoey/chat) |
```
Comment (fubuloubu) [inline SECURITY.md:48]:
```suggestion
| Contact                                                | Keybase                                                     |
| ------------------------------------------------------ | ----------------------------------------------------------- |
| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)             |
| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notpeopling2day/chat) |
| [@unparalleled-js](https://github.com/unparalleled-js) | [@Antazoey](https://keybase.io/antazoey/chat)              |
```

File: SECURITY.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+
+Please submit disclosures to **all** of the following main points of contact for security related issues according to the [initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact) and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details) guidelines.
+
+For all security related issues, ApeWorX has the following main points of contact:
+
+| Contact                                                | Keybase                                                     |
+| ------------------------------------------------------ | ----------------------------------------------------------- |
+| [@fubuloubu](https://github.com/fubuloubu)             | [@fubuloubu](https://keybase.io/fubuloubu/chat)             |
+| [@NotPeopling2day](https://github.com/NotPeopling2day) | [@NotPeopling2day](https://keybase.io/notpeopling2day/chat) |
+| [@unparalleled-js](https://github.com/unparalleled-js) | [@Antazoey](https://keybase.io/antazoey/chat)              |
```
Comment (fubuloubu) [inline SECURITY.md:48]:
```suggestion
| [@unparalleled-js](https://github.com/unparalleled-js) | [@Antazoey](https://keybase.io/antazoey/chat)               |
```

File: SECURITY.md (hunk 8 above)
Comment (fubuloubu) [reply SECURITY.md:3]:
```suggestion
This document describes the Security Policy for ApeWorX Ltd., including vulnerability disclosures for the Ape Framework, any of it's maintained plugins, or any packages we maintain used by those as dependencies (see [here](](https://github.com/search?q=org%3Aapeworx+-repo%3AApeWorX%2Fapeworx.github.io&type=Repositories))).
```

**Review by unparalleled-js — APPROVED** [review]

**Review by NotPeopling2day — APPROVED** [review]

**Pull request merged**